.github/workflows/backend.yml

```diff
@@ -101,47 +101,10 @@ jobs:
           cat clj-kondo-${CLJ_KONDO_VERSION}-linux-static-amd64.zip.sha256 >> SHA256sum.txt
           echo " clj-kondo-${CLJ_KONDO_VERSION}-linux-static-amd64.zip" >> SHA256sum.txt
           sha256sum -c SHA256sum.txt
-          unzip clj-kondo-${CLJ_KONDO_VERSION}-linux-static-amd64.zip
-      - run: ./clj-kondo --version
+          unzip -d /usr/local/bin clj-kondo-${CLJ_KONDO_VERSION}-linux-static-amd64.zip
+      - run: clj-kondo --version
       - name: Run clj-kondo
-        run: >-
-          ./clj-kondo
-          --config ./.clj-kondo/config.edn
-          --config-dir ./.clj-kondo
-          --parallel
-          --lint
-          ./src
-          ./test
-          ./enterprise/backend/src
-          ./enterprise/backend/test
-          ./shared/src
-          ./shared/test
-          ./modules/drivers/athena/src
-          ./modules/drivers/athena/test
-          ./modules/drivers/bigquery-cloud-sdk/src
-          ./modules/drivers/bigquery-cloud-sdk/test
-          ./modules/drivers/druid/src
-          ./modules/drivers/druid/test
-          ./modules/drivers/googleanalytics/src
-          ./modules/drivers/googleanalytics/test
-          ./modules/drivers/mongo/src
-          ./modules/drivers/mongo/test
-          ./modules/drivers/oracle/src
-          ./modules/drivers/oracle/test
-          ./modules/drivers/presto-jdbc/src
-          ./modules/drivers/presto-jdbc/test
-          ./modules/drivers/redshift/src
-          ./modules/drivers/redshift/test
-          ./modules/drivers/snowflake/src
-          ./modules/drivers/snowflake/test
-          ./modules/drivers/sparksql/src
-          ./modules/drivers/sparksql/test
-          ./modules/drivers/sqlite/src
-          ./modules/drivers/sqlite/test
-          ./modules/drivers/sqlserver/src
-          ./modules/drivers/sqlserver/test
-          ./modules/drivers/vertica/src
-          ./modules/drivers/vertica/test
+        run: ./bin/kondo.sh
```

Comment:
Is it OK to delegate this `./clj-kondo` command to `bin/kondo.sh`?

`bin/kondo.sh` lints the exact [same files ](https://github.com/metabase/metabase/blob/8348b23268779f7e0a7468d29f08e1de1dc9ec17/bin/kondo.sh#L15-L18) as these:

```sh
$ find modules/drivers shared enterprise/backend \
     -maxdepth 2 \
     -type d \
     -name src -or -name test
modules/drivers/bigquery-cloud-sdk/test
modules/drivers/bigquery-cloud-sdk/src
modules/drivers/sparksql/test
modules/drivers/sparksql/src
modules/drivers/oracle/test
modules/drivers/oracle/src
modules/drivers/snowflake/test
modules/drivers/snowflake/src
modules/drivers/sqlite/test
modules/drivers/sqlite/src
modules/drivers/redshift/test
modules/drivers/redshift/src
modules/drivers/googleanalytics/test
modules/drivers/googleanalytics/src
modules/drivers/druid/test
modules/drivers/druid/src
modules/drivers/sqlserver/test
modules/drivers/sqlserver/src
modules/drivers/presto-jdbc/test
modules/drivers/presto-jdbc/src
modules/drivers/mongo/test
modules/drivers/mongo/src
modules/drivers/vertica/test
modules/drivers/vertica/src
modules/drivers/athena/test
modules/drivers/athena/src
shared/test
shared/src
enterprise/backend/test
enterprise/backend/src
```

…though it now [also lints](https://github.com/metabase/metabase/blob/8348b23268779f7e0a7468d29f08e1de1dc9ec17/bin/kondo.sh#L21) `bin/`.